### ISSUE TITLE
assets.quite = true doesn't works

### ISSUE THREAD

**Issue by nallwhy** [rails/sprockets-rails#386]
heroku
rails 4.2.7.1
sprockets-rails 3.2.0

I added `config.assets.quiet = true` to `production.rb`.

And I checked middlewares by `rake middleware` and below is the result.
```
~ $ rake middleware
use Rack::Sendfile
use ActionDispatch::Static
use Rack::Lock
use #<ActiveSupport::Cache::Strategy::LocalCache::Middleware:0x007f401dae6118>
use Rack::Timeout
use Rack::Runtime
use Rack::MethodOverride
use ActionDispatch::RequestId
use RequestStore::Middleware
use Sprockets::Rails::QuietAssets
use Rails::Rack::Logger
use ActionDispatch::ShowExceptions
use ActionDispatch::DebugExceptions
use ActionDispatch::RemoteIp
use ActionDispatch::Callbacks
use ActiveRecord::ConnectionAdapters::ConnectionManagement
use ActiveRecord::QueryCache
use ActionDispatch::Cookies
use ActionDispatch::Session::CookieStore
use ActionDispatch::Flash
use ActionDispatch::ParamsParser
use Remotipart::Middleware
use Rack::Head
use Rack::ConditionalGet
use Rack::ETag
use Warden::Manager
use OmniAuth::Strategies::Facebook
use Rack::Pjax
run Trevari::Application.routes
```

I can still see the logs like below.
```
2016-12-31T02:06:31.328484+00:00 heroku[router]: at=info method=GET path="/apple-touch-icon-120x120.png" host=trevari-staging.herokuapp.com request_id=8629ed51-e264-4c8c-8f87-5b1b8c0c0c2b fwd="1.241.167.69" dyno=web.1 connect=0ms service=92ms status=404 bytes=1758
2016-12-31T02:06:31.250787+00:00 app[web.1]: source=rack-timeout id=8629ed51-e264-4c8c-8f87-5b1b8c0c0c2b wait=16ms timeout=20000ms state=ready
2016-12-31T02:06:31.252302+00:00 app[web.1]: Started GET "/apple-touch-icon-120x120.png" for 1.241.167.69 at 2016-12-31 02:06:31 +0000
2016-12-31T02:06:31.260656+00:00 app[web.1]: source=rack-timeout id=8629ed51-e264-4c8c-8f87-5b1b8c0c0c2b wait=16ms timeout=20000ms service=10ms state=active
2016-12-31T02:06:31.338092+00:00 app[web.1]:
2016-12-31T02:06:31.338096+00:00 app[web.1]: ActionController::RoutingError (No route matches [GET] "/apple-touch-icon-120x120.png"):
2016-12-31T02:06:31.338097+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.7.1/lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
2016-12-31T02:06:31.338098+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.7.1/lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
2016-12-31T02:06:31.338098+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/railties-4.2.7.1/lib/rails/rack/logger.rb:38:in `call_app'
2016-12-31T02:06:31.338099+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/railties-4.2.7.1/lib/rails/rack/logger.rb:20:in `block in call'
2016-12-31T02:06:31.338100+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.7.1/lib/active_support/tagged_logging.rb:68:in `block in tagged'
2016-12-31T02:06:31.338101+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.7.1/lib/active_support/tagged_logging.rb:26:in `tagged'
2016-12-31T02:06:31.338101+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.7.1/lib/active_support/tagged_logging.rb:68:in `tagged'
2016-12-31T02:06:31.338102+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/railties-4.2.7.1/lib/rails/rack/logger.rb:20:in `call'
2016-12-31T02:06:31.338103+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/sprockets-rails-3.2.0/lib/sprockets/rails/quiet_assets.rb:13:in `call'
2016-12-31T02:06:31.338104+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/request_store-1.3.1/lib/request_store/middleware.rb:9:in `call'
2016-12-31T02:06:31.338104+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.7.1/lib/action_dispatch/middleware/request_id.rb:21:in `call'
2016-12-31T02:06:31.338105+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-1.6.5/lib/rack/methodoverride.rb:22:in `call'
2016-12-31T02:06:31.338106+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-1.6.5/lib/rack/runtime.rb:18:in `call'
2016-12-31T02:06:31.338107+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-timeout-0.4.2/lib/rack/timeout/core.rb:122:in `block in call'
2016-12-31T02:06:31.338107+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-timeout-0.4.2/lib/rack/timeout/support/timeout.rb:19:in `call'
2016-12-31T02:06:31.338108+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-timeout-0.4.2/lib/rack/timeout/support/timeout.rb:19:in `timeout'
2016-12-31T02:06:31.338109+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-timeout-0.4.2/lib/rack/timeout/core.rb:121:in `call'
2016-12-31T02:06:31.338110+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.7.1/lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
2016-12-31T02:06:31.338110+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.7.1/lib/action_dispatch/middleware/static.rb:120:in `call'
2016-12-31T02:06:31.338111+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-1.6.5/lib/rack/sendfile.rb:113:in `call'
2016-12-31T02:06:31.338112+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/railties-4.2.7.1/lib/rails/engine.rb:518:in `call'
2016-12-31T02:06:31.338112+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/railties-4.2.7.1/lib/rails/application.rb:165:in `call'
2016-12-31T02:06:31.338113+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/rack-cors-0.4.0/lib/rack/cors.rb:80:in `call'
2016-12-31T02:06:31.338114+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/configuration.rb:225:in `call'
2016-12-31T02:06:31.338114+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/server.rb:578:in `handle_request'
2016-12-31T02:06:31.338115+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/server.rb:415:in `process_client'
2016-12-31T02:06:31.338116+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/server.rb:275:in `block in run'
2016-12-31T02:06:31.338117+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/thread_pool.rb:116:in `call'
2016-12-31T02:06:31.338117+00:00 app[web.1]:   vendor/bundle/ruby/2.2.0/gems/puma-3.6.0/lib/puma/thread_pool.rb:116:in `block in spawn_thread'
```

What did I do wrong?

**Comment by rafaelfranca** [comment]
It is a bug in rails 4.2.7. It will be fixed in 4.2.8

**Comment by nallwhy** [comment]
@rafaelfranca Thanks. When it will be released?
